core/controllers/profile.py

```diff
@@ -532,6 +532,17 @@ def post(self):
 class SiteLanguageHandler(base.BaseHandler):
     """Changes the preferred system language in the user's preferences."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'PUT': {
+            'site_language_code': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            }
```

Comment:
I think we have a list of allowed languages somewhere, can we only allow these here.

Reply:
Hey @vojtechjelinek I have restricted the value of "site_language_code" to the the list of all allowed languages. Can you please verify them? Thank you